index.js

```diff
@@ -0,0 +1,3 @@
+require('babel-core/register')
+require('coffee-script/register')
+require('./app')
```

Comment:
Created this boot loader so that from the point of `./app` onwards we can write ES6.